polaris-configuration/polaris-configuration-client/src/main/java/com/tencent/polaris/configuration/client/internal/ConfigFilePersistHandler.java

```diff
@@ -0,0 +1,311 @@
+/*
+ * Tencent is pleased to support the open source community by making Polaris available.
+ *
+ * Copyright (C) 2019 THL A29 Limited, a Tencent company. All rights reserved.
+ *
+ * Licensed under the BSD 3-Clause License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.tencent.polaris.configuration.client.internal;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.RandomAccessFile;
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.nio.channels.FileChannel;
+import java.nio.channels.FileLock;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
+import com.tencent.polaris.api.config.verify.DefaultValues;
+import com.tencent.polaris.client.api.SDKContext;
+import com.tencent.polaris.client.util.NamedThreadFactory;
+import com.tencent.polaris.client.util.Utils;
+import com.tencent.polaris.api.plugin.configuration.ConfigFile;
+import com.tencent.polaris.factory.util.FileUtils;
+import com.tencent.polaris.logging.LoggerFactory;
+import org.slf4j.Logger;
+import org.yaml.snakeyaml.Yaml;
+
+import static com.tencent.polaris.api.config.verify.DefaultValues.PATTERN_CONFIG_FILE;
+import static java.nio.file.StandardCopyOption.ATOMIC_MOVE;
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
+
+/**
+ * 配置文件持久化处理器
+ *
+ * @author rod.xu
+ * @date 2022/09/26
+ */
+public class ConfigFilePersistHandler {

```

Comment:
Would it be better to rename it `ConfigFilePersistentHandler` ?

Reply:
ok